jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -131,6 +143,14 @@ public Configuration(
       String serviceName,
       SamplerConfiguration samplerConfig,
       ReporterConfiguration reporterConfig) {
+    this(serviceName, samplerConfig, reporterConfig, true);
```

Comment:
didn't we want to default this to `disableGlobalTracer = false`?

also, let's stop adding constructors and add a builder, which we should've done in the first place. The new constructor below should be private.

Reply:
While I agree, I think it should be part of a separate commit